.github/workflows/code-checker.yml

```diff
@@ -74,6 +74,8 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: returntocorp/semgrep@sha256:ffc6f3567654f9431456d49fd059dfe548f007c494a7eb6cd5a1a3e50d813fb3
+    # Skip any PR created by dependabot and any check triggered by merge group
+    if: (github.actor != 'dependabot[bot]') && (github.event != 'merge_group')
```

Comment:
Not sure I understand the latter half of this. Running on dependabot does seem wasteful (except that we'd want to bump the semgrep hash automagically, perhaps, and would prefer it runs then?) but would we ever have a scenario where PR A passes semgrep and PR B passes semgrep but the combination A+B doesn't? I guess it probably depends on the rules? My 2c., but I'd perhaps leave off the latter.

Reply:
Per the semgrep project, merge queues are not supported at the moment. This is their workaround to allow the group to function.

Reply:
Ah, this is sad!